web/src/pages/people/index.tsx

```diff
@@ -44,9 +44,6 @@ export default function PeoplePage() {
       setIsLoading(false);
     }
   };
-  useEffect(() => {
```

Comment:
Some questions about how much of the functionality we want to remove:
- should I _remove_ all the fetching calls at this stage since they won't be needed until projects has been implemented, or just disable them?
- on the backend, should I remove the functions for getting all users/all approved researchers or is there motive to keep them for direct API calls?

Reply:
On the frontend, if the initial fetching of people isn't needed anymore then i'd remove it. It can always be brought back from the git history if needed. We only need to run the fetch on a search now right?

For the backend, my feeling would be the same i.e. remove anything that isn't being used now as it can be reinstated later. But happy for you to wait for Tom's thoughts too if desired.